pkg/dhcp6client/dhcp6client.go

```diff
@@ -92,7 +96,7 @@ func (p *buffer) remaining() []byte {
 
 func (c *Client) ReadReply() (*dhcp6.Packet, error) {
 	var err error
-	for i := 0; i < 5; i++ { // five attempts
+	for i := 0; i < c.attempts; i++ { // five attempts
 		pb := make([]byte, 1500)
```

Comment:
It is not, I don't think 576 is right. Besides the fact that 576 is probably impractical (this spec is from 1981), IP packets have up to 16 bit length, as noted there.

1500 is the number of bytes that mdlayher's dhcp6 server accepts. I don't know where it comes from, I assume it's an IPv6 header + UDP header + max dhcp6 packet len.